<!DOCTYPE html>
<html lang="en" dir="auto">

<head><meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
<meta name="robots" content="index, follow">
<title>Generic residue numbering of the GAIN domain of adhesion GPCRs | Guillermo Pérez-Hernández</title>
<meta name="keywords" content="">
<meta name="description" content=" The GPCR autoproteolysis inducing (GAIN) domain is an ancient protein fold ubiquitous in adhesion G protein-coupled receptors (aGPCR). It contains a tethered agonist necessary and sufficient for receptor activation. The GAIN domain is a hotspot for pathological mutations. However, the low primary sequence conservation of GAIN domains has thus far hindered the knowledge transfer across different GAIN domains in human receptors as well as species orthologs. Here, we present a scheme for generic residue numbering of GAIN domains, based on structural alignments of over 14,000 modeled GAIN domain structures. This scheme is implemented in the GPCR database (GPCRdb) and elucidates the domain topology across different aGPCRs and their homologs in a large panel of species. We identify conservation hotspots and statistically cancer-enriched positions in human aGPCRs and show the transferability of positional and structural information between GAIN domain homologs. The GAIN-GRN scheme provides a robust strategy to allocate structural homologies at the primary and secondary levels also to GAIN domains of polycystic kidney disease 1/PKD1-like proteins, which now renders positions in both GAIN domain types comparable to one another. In summary, our work enables researchers to generate hypothesis and rationalize experiments related to GAIN domain function and pathology.">
<meta name="author" content="Florian Seufert,&thinsp;Guillermo Pérez-Hernández,&thinsp;Gáspár Pándy-Szekeres,&thinsp;Ramon Guixà-González,&thinsp;Tobias Langenhan,&thinsp;David E. Gloriam,&thinsp;Peter W. Hildebrand">
<link rel="canonical" href="https://gph82.github.io/papers/article_26/">
<link crossorigin="anonymous" href="/assets/css/stylesheet.1d83721b120bc732b5bc8ea3ce28dc3f80e6d5d40345462ce6bae19959f4fd0f.css" integrity="sha256-HYNyGxILxzK1vI6jzijcP4Dm1dQDRUYs5rrhmVn0/Q8=" rel="preload stylesheet" as="style">
<link rel="icon" href="https://gph82.github.io/favicon.ico">
<link rel="icon" type="image/png" sizes="16x16" href="https://gph82.github.io/favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="https://gph82.github.io/favicon-32x32.png">
<link rel="apple-touch-icon" href="https://gph82.github.io/apple-touch-icon.png">
<meta name="theme-color" content="#2e2e33">
<meta name="msapplication-TileColor" content="#2e2e33">
<link rel="alternate" hreflang="en" href="https://gph82.github.io/papers/article_26/">
<noscript>
    <style>
        #theme-toggle,
        .top-link {
            display: none;
        }

    </style>
</noscript><meta property="og:url" content="https://gph82.github.io/papers/article_26/">
  <meta property="og:site_name" content="Guillermo Pérez-Hernández">
  <meta property="og:title" content="Generic residue numbering of the GAIN domain of adhesion GPCRs">
  <meta property="og:description" content=" The GPCR autoproteolysis inducing (GAIN) domain is an ancient protein fold ubiquitous in adhesion G protein-coupled receptors (aGPCR). It contains a tethered agonist necessary and sufficient for receptor activation. The GAIN domain is a hotspot for pathological mutations. However, the low primary sequence conservation of GAIN domains has thus far hindered the knowledge transfer across different GAIN domains in human receptors as well as species orthologs. Here, we present a scheme for generic residue numbering of GAIN domains, based on structural alignments of over 14,000 modeled GAIN domain structures. This scheme is implemented in the GPCR database (GPCRdb) and elucidates the domain topology across different aGPCRs and their homologs in a large panel of species. We identify conservation hotspots and statistically cancer-enriched positions in human aGPCRs and show the transferability of positional and structural information between GAIN domain homologs. The GAIN-GRN scheme provides a robust strategy to allocate structural homologies at the primary and secondary levels also to GAIN domains of polycystic kidney disease 1/PKD1-like proteins, which now renders positions in both GAIN domain types comparable to one another. In summary, our work enables researchers to generate hypothesis and rationalize experiments related to GAIN domain function and pathology.">
  <meta property="og:locale" content="en">
  <meta property="og:type" content="article">
    <meta property="article:section" content="papers">
    <meta property="article:published_time" content="2025-01-02T00:00:00+00:00">
    <meta property="article:modified_time" content="2025-01-02T00:00:00+00:00">
    <meta property="og:image" content="https://gph82.github.io/paper1.png">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:image" content="https://gph82.github.io/paper1.png">
<meta name="twitter:title" content="Generic residue numbering of the GAIN domain of adhesion GPCRs">
<meta name="twitter:description" content=" The GPCR autoproteolysis inducing (GAIN) domain is an ancient protein fold ubiquitous in adhesion G protein-coupled receptors (aGPCR). It contains a tethered agonist necessary and sufficient for receptor activation. The GAIN domain is a hotspot for pathological mutations. However, the low primary sequence conservation of GAIN domains has thus far hindered the knowledge transfer across different GAIN domains in human receptors as well as species orthologs. Here, we present a scheme for generic residue numbering of GAIN domains, based on structural alignments of over 14,000 modeled GAIN domain structures. This scheme is implemented in the GPCR database (GPCRdb) and elucidates the domain topology across different aGPCRs and their homologs in a large panel of species. We identify conservation hotspots and statistically cancer-enriched positions in human aGPCRs and show the transferability of positional and structural information between GAIN domain homologs. The GAIN-GRN scheme provides a robust strategy to allocate structural homologies at the primary and secondary levels also to GAIN domains of polycystic kidney disease 1/PKD1-like proteins, which now renders positions in both GAIN domain types comparable to one another. In summary, our work enables researchers to generate hypothesis and rationalize experiments related to GAIN domain function and pathology.">


<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BreadcrumbList",
  "itemListElement": [
    {
      "@type": "ListItem",
      "position":  1 ,
      "name": "Publications",
      "item": "https://gph82.github.io/papers/"
    }, 
    {
      "@type": "ListItem",
      "position":  2 ,
      "name": "Generic residue numbering of the GAIN domain of adhesion GPCRs",
      "item": "https://gph82.github.io/papers/article_26/"
    }
  ]
}
</script>
<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "BlogPosting",
  "headline": "Generic residue numbering of the GAIN domain of adhesion GPCRs",
  "name": "Generic residue numbering of the GAIN domain of adhesion GPCRs",
  "description": " The GPCR autoproteolysis inducing (GAIN) domain is an ancient protein fold ubiquitous in adhesion G protein-coupled receptors (aGPCR). It contains a tethered agonist necessary and sufficient for receptor activation. The GAIN domain is a hotspot for pathological mutations. However, the low primary sequence conservation of GAIN domains has thus far hindered the knowledge transfer across different GAIN domains in human receptors as well as species orthologs. Here, we present a scheme for generic residue numbering of GAIN domains, based on structural alignments of over 14,000 modeled GAIN domain structures. This scheme is implemented in the GPCR database (GPCRdb) and elucidates the domain topology across different aGPCRs and their homologs in a large panel of species. We identify conservation hotspots and statistically cancer-enriched positions in human aGPCRs and show the transferability of positional and structural information between GAIN domain homologs. The GAIN-GRN scheme provides a robust strategy to allocate structural homologies at the primary and secondary levels also to GAIN domains of polycystic kidney disease 1/PKD1-like proteins, which now renders positions in both GAIN domain types comparable to one another. In summary, our work enables researchers to generate hypothesis and rationalize experiments related to GAIN domain function and pathology.",
  "keywords": [
    
  ],
  "articleBody": "Fig. 5: Cancer Genome Atlas mutations mapped onto the GAIN domain via Generic Reisdue Numbers show mutational hotspots DOI 10.1038/s41467-024-55466-6 Abstract The GPCR autoproteolysis inducing (GAIN) domain is an ancient protein fold ubiquitous in adhesion G protein-coupled receptors (aGPCR). It contains a tethered agonist necessary and sufficient for receptor activation. The GAIN domain is a hotspot for pathological mutations. However, the low primary sequence conservation of GAIN domains has thus far hindered the knowledge transfer across different GAIN domains in human receptors as well as species orthologs. Here, we present a scheme for generic residue numbering of GAIN domains, based on structural alignments of over 14,000 modeled GAIN domain structures. This scheme is implemented in the GPCR database (GPCRdb) and elucidates the domain topology across different aGPCRs and their homologs in a large panel of species. We identify conservation hotspots and statistically cancer-enriched positions in human aGPCRs and show the transferability of positional and structural information between GAIN domain homologs. The GAIN-GRN scheme provides a robust strategy to allocate structural homologies at the primary and secondary levels also to GAIN domains of polycystic kidney disease 1/PKD1-like proteins, which now renders positions in both GAIN domain types comparable to one another. In summary, our work enables researchers to generate hypothesis and rationalize experiments related to GAIN domain function and pathology. ",
  "wordCount" : "219",
  "inLanguage": "en",
  "image":"https://gph82.github.io/paper1.png","datePublished": "2025-01-02T00:00:00Z",
  "dateModified": "2025-01-02T00:00:00Z",
  "author":[{
    "@type": "Person",
    "name": "Florian Seufert"
  }, {
    "@type": "Person",
    "name": "\u003cb\u003eGuillermo Pérez-Hernández\u003c/b\u003e"
  }, {
    "@type": "Person",
    "name": "Gáspár Pándy-Szekeres"
  }, {
    "@type": "Person",
    "name": "Ramon Guixà-González"
  }, {
    "@type": "Person",
    "name": "Tobias Langenhan"
  }, {
    "@type": "Person",
    "name": "David E. Gloriam"
  }, {
    "@type": "Person",
    "name": "Peter W. Hildebrand"
  }],
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://gph82.github.io/papers/article_26/"
  },
  "publisher": {
    "@type": "Organization",
    "name": "Guillermo Pérez-Hernández",
    "logo": {
      "@type": "ImageObject",
      "url": "https://gph82.github.io/favicon.ico"
    }
  }
}
</script>



<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/katex.min.css" integrity="sha384-wcIxkf4k558AjM3Yz3BBFQUbk/zgIYC2R0QpeeYb+TwlBVMrlgLqwRjRtGZiK7ww" crossorigin="anonymous">
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/katex.min.js" integrity="sha384-hIoBPJpTUs74ddyc4bFZSM1TVlQDA60VBbJS0oA934VSz82sBx1X7kSx2ATBDIyd" crossorigin="anonymous"></script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.16.10/dist/contrib/auto-render.min.js" integrity="sha384-43gviWU0YVjaDtb/GhzOouOXtZMP/7XUzwPTstBeZFe/+rCMvRwr4yROQP43s0Xk" crossorigin="anonymous"
  onload="renderMathInElement(document.body);"></script>

<script>
    document.addEventListener("DOMContentLoaded", function() {
        renderMathInElement(document.body, {
          delimiters: [
            {left: '$$', right: '$$', display: true},
            {left: '$', right: '$', display: false},
            {left: "\\begin{equation}", right: "\\end{equation}", display: true},
            {left: "\\begin{equation*}", right: "\\end{equation*}", display: true},
            {left: "\\begin{align}", right: "\\end{align}", display: true},
            {left: "\\begin{align*}", right: "\\end{align*}", display: true},
            {left: "\\begin{alignat}", right: "\\end{alignat}", display: true},
            {left: "\\begin{gather}", right: "\\end{gather}", display: true},
            {left: "\\begin{CD}", right: "\\end{CD}", display: true},
          ],
          throwOnError : false
        });
    });
</script>
 


</head>

<body class="" id="top">

<header class="header">
    <nav class="nav">
        <div class="logo">
            <a href="https://gph82.github.io/" accesskey="h" title=" "> </a>
            <div class="logo-switches">
            </div>
        </div>
        <ul id="menu">
            <li>
                <a href="https://gph82.github.io/" title="home">
                    <span>
                          <svg xmlns="http://www.w3.org/2000/svg" width="1em" height="1em" style="vertical-align: text-bottom;" viewBox="0 0 576 512" fill="currentColor">
                            <path d="M575.8 255.5c0 18-15 32.1-32 32.1h-32v160c0 22.1-17.9 40-40 40h-48V352c0-17.7-14.3-32-32-32H184c-17.7 0-32 14.3-32 32v135.6H104c-22.1 0-40-17.9-40-40V287.6H32c-18 0-32-14.1-32-32.1 0-9 3-17 10-24L266.5 7c13.3-13.3 34.9-13.3 48.2 0L565.8 231.5c8 7 10 15 10 24z"/>
                          </svg>
                    </span>
                </a>
            </li>
            <li>
                <a href="https://gph82.github.io/papers/" title="Publications">
                    <span>Publications
                    </span>
                </a>
            </li>
            <li>
                <a href="https://gph82.github.io/software/" title="Software">
                    <span>Software
                    </span>
                </a>
            </li>
            <li>
                <a href="https://gph82.github.io/affiliation/" title="Affiliation">
                    <span>Affiliation
                    </span>
                </a>
            </li>
            <li>
                <a href="https://gph82.github.io/outreach/" title="Outreach">
                    <span>Outreach
                    </span>
                </a>
            </li>
            <li>
                <a href="https://gph82.github.io/scipol/" title="CERFA &amp; RAICEX">
                    <span>CERFA &amp; RAICEX
                    </span>
                </a>
            </li>
            <li>
                <a href="https://gph82.github.io/CV.pdf" title="CV">
                    <span>CV
                    </span>
                </a>
            </li>
        </ul>
    </nav>
</header>

    <main class="main">

<article class="post-single">
  <header class="post-header">
    
    <h1 class="post-title entry-hint-parent">
      Generic residue numbering of the GAIN domain of adhesion GPCRs
    </h1>
    <div class="post-meta"><span title='2025-01-02 00:00:00 +0000 UTC'>January 2025</span>&nbsp;&middot;&nbsp;Florian Seufert,&thinsp;<b>Guillermo Pérez-Hernández</b>,&thinsp;Gáspár Pándy-Szekeres,&thinsp;Ramon Guixà-González,&thinsp;Tobias Langenhan,&thinsp;David E. Gloriam,&thinsp;Peter W. Hildebrand&nbsp;&middot;&nbsp;<a href="https://doi.org/10.1038/s41467-024-55466-6" rel="noopener noreferrer" target="_blank">Nature Communications</a>

</div>
  </header> 
  <div class="post-content"><h5 id="fig-5-cancer-genome-atlas-mutations-mapped-onto-the-gain-domain-via-generic-reisdue-numbers-show-mutational-hotspots">Fig. 5: Cancer Genome Atlas mutations mapped onto the GAIN domain via Generic Reisdue Numbers show mutational hotspots<a hidden class="anchor" aria-hidden="true" href="#fig-5-cancer-genome-atlas-mutations-mapped-onto-the-gain-domain-via-generic-reisdue-numbers-show-mutational-hotspots">#</a></h5>
<h2><img loading="lazy" src="/papers/article_26/paper1.png"></h2>
<h5 id="doi">DOI<a hidden class="anchor" aria-hidden="true" href="#doi">#</a></h5>
<h2 id="101038s41467-024-55466-6"><a href="https://doi.org/10.1038/s41467-024-55466-6" target="_blank">10.1038/s41467-024-55466-6</a><a hidden class="anchor" aria-hidden="true" href="#101038s41467-024-55466-6">#</a></h2>
<h5 id="abstract">Abstract<a hidden class="anchor" aria-hidden="true" href="#abstract">#</a></h5>
<pre><code>              The GPCR autoproteolysis inducing (GAIN) domain is an ancient protein fold ubiquitous in adhesion G protein-coupled receptors (aGPCR). It contains a tethered agonist necessary and sufficient for receptor activation. The GAIN domain is a hotspot for pathological mutations. However, the low primary sequence conservation of GAIN domains has thus far hindered the knowledge transfer across different GAIN domains in human receptors as well as species orthologs. Here, we present a scheme for generic residue numbering of GAIN domains, based on structural alignments of over 14,000 modeled GAIN domain structures. This scheme is implemented in the GPCR database (GPCRdb) and elucidates the domain topology across different aGPCRs and their homologs in a large panel of species. We identify conservation hotspots and statistically cancer-enriched positions in human aGPCRs and show the transferability of positional and structural information between GAIN domain homologs. The GAIN-GRN scheme provides a robust strategy to allocate structural homologies at the primary and secondary levels also to GAIN domains of polycystic kidney disease 1/PKD1-like proteins, which now renders positions in both GAIN domain types comparable to one another. In summary, our work enables researchers to generate hypothesis and rationalize experiments related to GAIN domain function and pathology.
</code></pre>


  </div>

  <footer class="post-footer">
    <ul class="post-tags">
    </ul>
  </footer>
</article>
    </main>
    
<footer class="footer">
    <span>&copy; 2026 Guillermo Pérez-Hernández</span> ·     
    <span>
    Powered by 
        <a href="https://gohugo.io/" rel="noopener noreferrer" target="_blank">hugo</a>, <a href="https://github.com/adityatelange/hugo-PaperMod/" rel="noopener" target="_blank">papermod</a>, &amp;
        <a href="https://github.com/pmichaillat/hugo-website/" rel="noopener" target="_blank">hugo-website</a>
    </span> · 
    <a href='/impressum/' target="_blank" rel="noopener">Impressum</a>
<a href="#top" aria-label="go to top" title="Go to Top (Alt + G)" class="top-link" id="top-link" accesskey="g">
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 12 6" fill="currentColor">
        <path d="M12 6H0l6-6z" />
    </svg>
</a>

<script>
    let menu = document.getElementById('menu')
    if (menu) {
        menu.scrollLeft = localStorage.getItem("menu-scroll-position");
        menu.onscroll = function () {
            localStorage.setItem("menu-scroll-position", menu.scrollLeft);
        }
    }

    document.querySelectorAll('a[href^="#"]').forEach(anchor => {
        anchor.addEventListener("click", function (e) {
            e.preventDefault();
            var id = this.getAttribute("href").substr(1);
            if (!window.matchMedia('(prefers-reduced-motion: reduce)').matches) {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView({
                    behavior: "smooth"
                });
            } else {
                document.querySelector(`[id='${decodeURIComponent(id)}']`).scrollIntoView();
            }
            if (id === "top") {
                history.replaceState(null, null, " ");
            } else {
                history.pushState(null, null, `#${id}`);
            }
        });
    });

</script>
<script>
    var mybutton = document.getElementById("top-link");
    window.onscroll = function () {
        if (document.body.scrollTop > 800 || document.documentElement.scrollTop > 800) {
            mybutton.style.visibility = "visible";
            mybutton.style.opacity = "1";
        } else {
            mybutton.style.visibility = "hidden";
            mybutton.style.opacity = "0";
        }
    };

</script>
<script>
    document.querySelectorAll('pre > code').forEach((codeblock) => {
        const container = codeblock.parentNode.parentNode;

        const copybutton = document.createElement('button');
        copybutton.classList.add('copy-code');
        copybutton.innerHTML = 'copy';

        function copyingDone() {
            copybutton.innerHTML = 'copied!';
            setTimeout(() => {
                copybutton.innerHTML = 'copy';
            }, 2000);
        }

        copybutton.addEventListener('click', (cb) => {
            if ('clipboard' in navigator) {
                navigator.clipboard.writeText(codeblock.textContent);
                copyingDone();
                return;
            }

            const range = document.createRange();
            range.selectNodeContents(codeblock);
            const selection = window.getSelection();
            selection.removeAllRanges();
            selection.addRange(range);
            try {
                document.execCommand('copy');
                copyingDone();
            } catch (e) { };
            selection.removeRange(range);
        });

        if (container.classList.contains("highlight")) {
            container.appendChild(copybutton);
        } else if (container.parentNode.firstChild == container) {
            
        } else if (codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.nodeName == "TABLE") {
            
            codeblock.parentNode.parentNode.parentNode.parentNode.parentNode.appendChild(copybutton);
        } else {
            
            codeblock.parentNode.appendChild(copybutton);
        }
    });
</script>
</body>
</html>
